### PR TITLE
Update to include the endpoint clients.config.office.net for M365

### PR DESCRIPTION
**Pull request by umair0204**
WsyncMgr fails to sync Office updates if the endpoint is not included in the Proxy/firewall settings.

### CODE REVIEW

File: memdocs/configmgr/core/plan-design/network/internet-endpoints.md [hunk 1]
```diff
@@ -182,6 +182,8 @@ If you use Configuration Manager to deploy and update Microsoft 365 Apps for ent
 
 - `contentstorage.osi.office.net` to support the evaluation of Office add-in readiness<!-- MEMDocs#410 -->
 
+- `clients.config.office.net` used to retrieve the names of the files needed for a particular Microsoft 365 Apps update. For more information, see [Using the Microsoft 365 Apps file list API](https://learn.microsoft.com/en-us/office/client-developer/shared/manageability-applications-with-the-office-365-click-to-run-installer#using-the-microsoft-365-apps-file-list-api).
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/core/plan-design/network/internet-endpoints.md:185]:
```suggestion
- `clients.config.office.net` to retrieve the names of the files needed for a particular Microsoft 365 Apps update. For more information, see [Using the Microsoft 365 Apps file list API](/office/client-developer/shared/manageability-applications-with-the-office-365-click-to-run-installer#using-the-microsoft-365-apps-file-list-api).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @umair0204, kindly commit the suggested changes. Many thanks.

**Comment by traya1** [comment]
Learn Build status updates of commit _[b3916d0](https://github.com/umair0204/memdocs/commits/b3916d0b9f92d917fad1e922982d59f62439c8e6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/configmgr/core/plan-design/network/internet-endpoints.md](https://github.com/umair0204/memdocs/blob/patch-1/memdocs/configmgr/core/plan-design/network/internet-endpoints.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212081843197699-3385/BuildReport?accessString=affca7acce51346bf84df4a99576586bb2cc6a3307c8c926c9beac9e54824ed5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dougeby** [comment]
approved

**Pull request merged**